associations/serializers/library.py

```diff
@@ -103,10 +105,21 @@ class LoanableSerializer(serializers.ModelSerializer):
     library = serializers.PrimaryKeyRelatedField(
         many=False, read_only=False, queryset=Library.objects
     )
+    tags = serializers.SerializerMethodField
 
     class Meta:
         model = Loanable
-        fields = ("id", "name", "description", "image", "comment", "library")
+        read_only_fields = ("id", "tags")
+        fields = read_only_fields + (
+            "name",
+            "description",
+            "image",
+            "comment",
+            "library",
+        )
+
+    def get_tags(self, obj):
```

Comment:
Maybe create a `TagSerializerField` and make `tags = TagSerializerField` to avoid copy pasting the code in each serializer

Reply:
Well I tried, but the problem is that we need to have access to the `context` (aka the `user`) in this `Field`. And I don't see any obvious way to do it.
* The `Fields` do not seem to have a `context` variable automatically passed.
* We can always pass one to their constructor, but the place where the `Fields` are declared in a `Serializer` does not have access to the `context`.

Reply:
Maybe pass it as a constructor argument